src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -42,14 +42,21 @@ class SgeBackend extends Backend with SharedFileSystem with LazyLogging {
         launchQsub(backendCall) match {
           case (qsubRc, _) if qsubRc != 0 => FailedExecution(new Throwable(s"Error: qsub exited with return code: $qsubRc"))
           case (_, None) => FailedExecution(new Throwable(s"Could not parse Job ID from qsub output"))
-          case (_, Some(sgeJobId)) => pollForSgeJobCompletionThenPostProcess(backendCall, sgeJobId)
+          case (_, Some(sgeJobId)) =>
+            updateSgeJobTable(backendCall, "Running", None, Option(sgeJobId))
+            pollForSgeJobCompletionThenPostProcess(backendCall, sgeJobId)
         }
       case Failure(ex) => FailedExecution(ex)
     }
   }
 
+  private def updateSgeJobTable(call: BackendCall, status: String, rc: Option[Int], sgeJobId: Option[Int]) = {
+    val backendInfo = SgeCallBackendInfo(CallStatus(status, rc), sgeJobId)
+    DataAccess.instance.updateExecutionBackendInfo(call.workflowDescriptor.id, call.call, backendInfo)
```

Comment:
I guess I should at least compose the two DB updates so they can't happen in the wrong order... But otherwise what can you do if the status update fails? I don't want to fail the workflow...